test/WebSites/ValidationWebSite/ProductValidatorAttibute.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.ComponentModel.DataAnnotations;
+using ValidationWebSite.ViewModels;
+
+namespace ValidationWebSite
+{
+    public class ProductValidatorAttribute : ValidationAttribute
+    {
+        protected override ValidationResult IsValid(object value, ValidationContext validationContext)
+        {
+
+            if (value is ProductViewModel)
+            {
+                var product = (ProductViewModel)value;
+                if (!product.Country.Equals("USA") && string.IsNullOrEmpty(product.Name))
```

Comment:
not clear why products need to be made in the USA only if they aren't named.  should this condition use `||`?